src/app/containers/Timestamp/index.test.jsx

```diff
@@ -1,25 +1,68 @@
 import React from 'react';
+import { render } from 'enzyme';
 import { isNull, shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
+import timestampGenerator from './helpers/timestampGenerator';
 import Timestamp from '.';
 
 const timestamp = 1539969006000; // 19 October 2018
 const noLeadingZeroTimestamp = 1530947227000; // 07 July 2018
 const invalidTimestamp = 8640000000000001; // A day holds 86,400,000 milliseconds - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#Description
 
+const eighthMarch = 1552009884472; // 2019-03-08T01:51:24.472Z
+const fifthJan = 1546707084472; // 2019-01-05T16:51:24.472Z
```

Comment:
Can we swap lines 11 and 12? Makes sense to have the earlier one before the later one :) saves a few seconds of "huh? 🤔"